src/index.tsx

```diff
@@ -13,6 +16,11 @@ import type {
 import type { ITooltipController } from './components/TooltipController/TooltipControllerTypes'
 import type { ITooltipWrapper } from './components/TooltipProvider/TooltipProviderTypes'
 
+// this content will be replaced in build time with the `react-tooltip.css` builded content
+const TooltipStyles = 'react-tooltip-css-placeholder'
+
+injectStyle(TooltipStyles)
```

Comment:
I didn't mean to commit it, but why was `@ts-ignore` being used here? No warnings or errors on my environment.

Reply:
yeap, all good! thanks!